script/tool/test/publish_check_command_test.dart

```diff
@@ -105,7 +105,8 @@ void main() {
 
       processRunner.processesToReturn.add(process);
 
-      expect(runner.run(<String>['publish-check']), completes);
+      expect(runner.run(<String>['publish-check', '--allow-pre-release']),
+          completes);
     });
```

Comment:
We also need a new test that the same setup fails without `--allow-pre-release`

Reply:
done